2019-02-24 19:02:27.596  WARN 16996 --- [main] o.m.s.mapper.ClassPathMapperScanner      : No MyBatis mapper was found in '[com.jack.hhitseat]' package. Please check your configuration.
2019-02-24 19:02:27.717  WARN 16996 --- [main] o.m.s.mapper.ClassPathMapperScanner      : No MyBatis mapper was found in '[com.aisino.arrms.mapper]' package. Please check your configuration.
2019-02-24 19:02:29.947  WARN 16996 --- [main] o.s.w.c.s.GenericWebApplicationContext   : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [com/jack/hhitseat/config/DruidConfiguration.class]: Invocation of init method failed; nested exception is java.sql.SQLException: url not set
2019-02-24 19:03:24.880  WARN 3652 --- [restartedMain] o.m.s.mapper.ClassPathMapperScanner      : No MyBatis mapper was found in '[com.jack.hhitseat]' package. Please check your configuration.
2019-02-24 19:03:24.990  WARN 3652 --- [restartedMain] o.m.s.mapper.ClassPathMapperScanner      : No MyBatis mapper was found in '[com.aisino.arrms.mapper]' package. Please check your configuration.
2019-02-24 19:03:26.530  WARN 3652 --- [restartedMain] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.context.ApplicationContextException: Unable to start web server; nested exception is org.springframework.boot.web.server.WebServerException: Unable to start embedded Tomcat
2019-02-24 19:20:35.292  WARN 14892 --- [main] o.m.s.mapper.ClassPathMapperScanner      : No MyBatis mapper was found in '[com.jack.hhitseat]' package. Please check your configuration.
2019-02-24 19:20:35.405  WARN 14892 --- [main] o.m.s.mapper.ClassPathMapperScanner      : No MyBatis mapper was found in '[com.aisino.arrms.mapper]' package. Please check your configuration.
2019-02-24 19:20:37.169  WARN 14892 --- [main] o.s.w.c.s.GenericWebApplicationContext   : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [com/jack/hhitseat/config/DruidConfiguration.class]: Invocation of init method failed; nested exception is java.sql.SQLException: url not set
2019-02-24 19:23:03.552  WARN 16328 --- [main] o.m.s.mapper.ClassPathMapperScanner      : No MyBatis mapper was found in '[com.jack.hhitseat]' package. Please check your configuration.
2019-02-24 19:23:03.767  WARN 16328 --- [main] o.m.s.mapper.ClassPathMapperScanner      : No MyBatis mapper was found in '[com.jack.hhitseat]' package. Please check your configuration.
2019-02-24 19:23:05.319  WARN 16328 --- [main] o.s.w.c.s.GenericWebApplicationContext   : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [com/jack/hhitseat/config/DruidConfiguration.class]: Invocation of init method failed; nested exception is java.sql.SQLException: url not set
